src/main/java/org/spongepowered/mod/event/SpongeForgeEventFactory.java

```diff
@@ -683,17 +683,17 @@ public static ChunkEvent createChunkEvent(Event event) {
         }
 
         net.minecraft.world.World forgeWorld = (net.minecraft.world.World) world.get();
-        Explosion explosion = (Explosion) spongeEvent.getExplosion();
+        Explosion explosion = (Explosion) null;//spongeEvent.getExplosion();
```

Comment:
I'd actually forgotten about this until now, I had meant to comment on it in the description - the `Explosion` is not available in the Sponge `ExplosionEvent` parent, only `Start`, `Detonate`, and `Post`. I'm not sure what to do here other than:
- passing `null` to the Forge event constructor
- throwing an exception if this method is called
 - Forge does not call this method natively, mods might, so:
- printing a warning to console and returning `null`